nvi-commons/src/main/java/no/sikt/nva/nvi/common/service/NviCandidateService.java

```diff
@@ -0,0 +1,41 @@
+package no.sikt.nva.nvi.common.service;
+
+import java.net.URI;
+import java.util.List;
+import java.util.Optional;
+import no.sikt.nva.nvi.common.NviCandidateRepository;
+import no.sikt.nva.nvi.common.model.business.ApprovalStatus;
+import no.sikt.nva.nvi.common.model.business.Candidate;
+import no.sikt.nva.nvi.common.model.business.Status;
+
+public class NviCandidateService {
+
+    private final NviCandidateRepository nviCandidateRepository;
+
+    public NviCandidateService(NviCandidateRepository nviCandidateRepository) {
+        this.nviCandidateRepository = nviCandidateRepository;
+    }
+
+    public Optional<Candidate> getCandidateByPublicationId(URI publicationId) {
+        return nviCandidateRepository.findByPublicationId(publicationId.toString());
+    }
+
+    public boolean exists(URI publicationId) {
+        return nviCandidateRepository.findByPublicationId(publicationId.toString()).isPresent();
+    }
+
+    public void createCandidateWithPendingInstitutionApprovals(URI publicationId, List<URI> institutionUri) {
```

Comment:
Will it be possible to create Approved NviCandidate? I guess no, maybe rename to createCandidate?